WooCommerce/build.gradle

```
@@ -74,7 +74,7 @@ android {
         if (project.hasProperty("versionName")) {
             versionName project.property("versionName")
         } else {
-            versionName "9.9-rc-1"
+            versionName "9.9-rc-2"
```

This is not supposed to be done manually.

In general, please don't do anything related to releases in a PR unless you're following a release scenario as a release manager.

```suggestion
            versionName "9.9-rc-1"
```